src/test/kotlin/com/dansplugins/fiefs/fief/TestFief.kt

```diff
@@ -0,0 +1,47 @@
+package com.dansplugins.fiefs.fief
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.util.*
+
+class TestFief {
+
+    @Test
+    fun testInitialization() {
+        val uuid = UUID.randomUUID()
+        val fief = Fief("testFief", uuid)
+        assertEquals("testFief", fief.getName())
+        assertEquals(uuid, fief.getOwnerUUID())
+        assertEquals(0, fief.getMembers().size)
+        assertEquals(fief.isMember(uuid), true)
```

Comment:
Order of arguments for assertEquals is expected followed by actual.

Reply:
Switched some arguments around in this class, this should be good now.